src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -506,6 +491,65 @@ private static StringBuilder fakeBody(final Request... dispositions)
                 .append(String.format("--%s--", BOUNDARY));
             return builder;
         }
+
+        /**
+         * A private inner class for the Request implementation.
```

Comment:
@sebing we don't need to repeat that it is private inner class, it's visible from the code. We need to explain  what it does instead.